src/views/mfa-verify/CallAndSMSForm.js

```diff
@@ -33,7 +33,7 @@ define(['okta', 'vendor/lib/q', 'views/shared/TextBox'], function (Okta, Q, Text
     },
 
     initialize: function () {
-      var self = this;
+      var form = this;
```

Comment:
any reason we renamed it? It doesn't affect anything, but just curious... we could leave it as it was